### PR TITLE
CTI addendum threat.feed.name, threat.feed.dashboard_id

### PR DESCRIPTION
**Pull request by ecezalp**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? yes
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/main/CONTRIBUTING.md)? yes
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/main/rfcs/README.md)? yes
- If submitting code/script changes, have you verified all tests pass locally using `make test`? yes
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes? yes
- Is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed. yes
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/main/CHANGELOG.next.md)? no

### CODE REVIEW

File: schemas/threat.yml [hunk 1]
```diff
@@ -259,6 +259,22 @@
...
+      example: 5ba16340-72e6-11eb-a3e3-b3cc7c78a70f
+
+    - name: feed.name
+      level: extended
+      type: keyword
+      short: Name of the threat feed.
+      description: >
+        The name of the threat feed in UI friendly format.
+      example: "AlienVault OTX"
+
```
Comment (peasead) [inline schemas/threat.yml:277]:
```suggestion

    - name: feed.description
      level: extended
      type: keyword
      short: Description of the threat feed.
      description: >
        Description of the threat feed in a UI friendly format.
      example: "Threat feed from the AlienVault Open Threat eXchange network."

    - name: feed.reference
      level: extended
      type: keyword
      short: Reference for the threat feed.
      description: >
        Reference information for the threat feed in a UI friendly format.
      example: "https://otx.alienvault.com"

File: schemas/threat.yml [hunk 2]
```diff
@@ -259,6 +259,37 @@
...
+      example: "Threat feed from the AlienVault Open Threat eXchange network."
+
+    - name: feed.reference
+      level: extended
+      type: keyword
+      short: Reference for the threat feed.
+      description: >
+        Reference information for the threat feed in a UI friendly format.
+      example: "https://otx.alienvault.com"
     - name: framework
```
Comment (peasead) [inline schemas/threat.yml:293]:
```suggestion

    - name: framework
```

**Review by peasead — APPROVED** [review]
LGTM

**Review by ebeahan — CHANGES_REQUESTED** [review]
Field additions look good to me!

CI is failing because there's several files that need updated each time the schema's updated.

If you run `make` from the top-level of the repo, it'll auto-generate all the necessary changes. After running `make`, add and commit the updates artifacts/docs and push the changes. This should make CI go ✅.

Finally, can you add a CHANGELOG.next entry under the 8.2 header: https://github.com/elastic/ecs/blob/main/CHANGELOG.next.md#820-soft-feature-freeze? Though 8.2 is under soft FF, there's no issue adding these fields.

**Review by ebeahan — APPROVED** [review]
LGTM. Thanks!

**Comment by ebeahan** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.2|[](https://github.com/elastic/ecs/pull/1851)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**